src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <Display />
+      <ButtonPanel />
+    </div>
```

Comment:
so you have two of these files hahaha
You need to delete one, please :) 

This should be the correct one (wrong one if comment above)